Update Networking intro and move Multiplayer Gem topics underneath (#2051)
o3de/o3de.org: content/docs/user-guide/networking/multiplayer/test-in-editor.md

---
linktitle: Testing in Editor

> title: Test Multiplayer Projects in the O3DE Editor
> description: Using Open 3D Engine Editor to test and edit synchronized multiplayer game projects.

Reviewer: ```suggestion
title: Testing Multiplayer Projects in O3DE Editor
description: Learn how to use O3DE Editor to test and edit synchronized multiplayer game projects in Open 3D Engine (O3DE).
```

weight: 500
---

> An important part of multiplayer simulation is getting a feel for how your client looks and performs when connected to a server. In order to help with that, Open 3D Engine offers the ability to automatically connect a client to a server. When in the **O3DE Editor**, as long as networked entities are on the scene and you have a current server launcher, entering Play Mode **(Ctrl+G)** will locally launch your server for you. As part of this process, the Editor will perform a one time send of all entities on your scene to the server so it can then load them. The Editor will then connect to the server as a client.

Reviewer: ```suggestion
An important part of multiplayer simulation is getting a feel for how your client looks and performs when connected to a server. To help with that, **Open 3D Engine (O3DE)** lets you automatically connect a client to a server. In **O3DE Editor**, if your networked entities are on the scene and you have a current server launcher, you can enter Play Mode (**Ctrl+G**) to locally launch your server. As part of this process, the Editor performs a one-time send of all entities on the scene to your server so that it can then load them. The Editor then connects to the server as a client.
```

> The nature of this feature has the Editor authoritating world state to the Server before ceding control to the server. This process is inherently risky and as such, this feature is compiled out for Release builds.

Reviewer: ```suggestion
The nature of this feature has the Editor authoritating world state to the Server before ceding control to the server. Because this process is inherently risky, this feature is compiled out for Release builds.
```

> ### Advanced Configuration

Reviewer: ```suggestion
### Advanced configuration
```

> The **Multiplayer Gem** exposes a variety of cvars to help configure launching a server when the Editor enters Play Mode.

Reviewer: ```suggestion
The **Multiplayer Gem** exposes a variety of cvars to help you configure how to launch a server when the Editor enters Play Mode.
```
